R/modelDown.R

```diff
@@ -109,6 +112,79 @@ modelDown <- function(...,
   }
 }
 
+validateParameters <- function(explainers, options, modules, output_folder, repository_name, should_open_website){
```

Comment:
`output_folder` and `repository_name` are ot used in validation, maybe remove them from arguments ?